concepts/80 Troubleshooting/50 A Drop-Down Editor Does Not Show Data/00 A Drop-Down Editor Does Not Show Data.md

```diff
@@ -0,0 +1,9 @@
+This section describes the issues related to the following drop-down editors:
+
+- [SelectBox](/Documentation/Guide/UI_Components/SelectBox/Getting_Started_with_SelectBox/)
+- [Lookup](/Documentation/Guide/UI_Components/Lookup/Getting_Started_with_Lookup/)
+- [DropDownBox](/Documentation/Guide/UI_Components/DropDownBox/Getting_Started_with_DropDownBox/)
+- [TagBox](/Documentation/Guide/UI_Components/TagBox/Getting_Started_with_TagBox/)
+- [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/)/[TreeList](/Documentation/Guide/UI_Components/TreeList/Getting_Started_with_TreeList/) lookup column editor
+
+Drop-down editors may not display data in different scenarios: on save, on first load, when the value property is changed in the code. They may also display different values, for example, IDs instead of text. Follow the steps below to learn how to troubleshoot your application. If the solutions listed here do not help, [create a ticket](https://supportcenter.devexpress.com/ticket/create) in our Support Center.
```

Comment:
Let's motivate a customer to add details: 
```suggestion
Drop-down editors may not display data in different scenarios: on save, on first load, when the value property is changed in the code. They may also display different values, for example, IDs instead of text. Follow the steps below to learn how to troubleshoot your application. If the solutions listed here do not help, create a ticket in our Support Center and describe your issue in details: [Create a ticket](https://supportcenter.devexpress.com/ticket/create).
```